README.md

```diff
@@ -157,6 +157,17 @@ There are rules from npm about what a package name can be. This is already enfor
 
 This requires manual fixing as automatically fixing this may lead to valid but incorrect package names.
 
+
+##### One scope for the monorepo, all packages are scoped
+
+> NOTE: These restrictions are not enforced
+
+To make publishing easier, it is useful if every package in a monorepo exists under a single scope. We strongly recommend picking a scope name to use for all your packages. If you have unscoped packages, or multiple scopes, you introduce a risk that someone who is publishing may not have all necessary publish permissions, leading to failed publishes. The only exceptions you should make to this is if a different tool such as `npx` requires a particular format to the package names.
```

Comment:
I don’t think npx cares about the package name, I think it’s only yarn create/npm init